network/mode/bridge/bridge.go

```diff
@@ -9,9 +9,9 @@ import (
 
 	"github.com/alibaba/pouch/apis/types"
 	"github.com/alibaba/pouch/daemon/mgr"
-	"github.com/alibaba/pouch/extra/libnetwork/Godeps/_workspace/src/github.com/Sirupsen/logrus"
 	"github.com/alibaba/pouch/network"
 	"github.com/alibaba/pouch/pkg/errtypes"
+	"github.com/sirupsen/logrus"
```

Comment:
And please move this line to L15, since this is a third-party package. We have a rule of how to import packages in : 

> When importing packages, to improve readabilities, we should import package by sequence: system packages, project's own packages and third-party packages. And we should keep a blank line among these three kinds of packages;

In https://github.com/alibaba/pouch/blob/master/docs/contributions/code_styles.md#additional-style-rules